python/graphscope/learning/glt_neighbor_loader.py

```diff
@@ -0,0 +1,53 @@
+from typing import Any
+from typing import Dict
+from typing import Iterator
+from typing import List
+from typing import Optional
+from typing import Tuple
+from typing import Union
+
+from torch_geometric.data import Data
+from torch_geometric.data import FeatureStore
+from torch_geometric.data import GraphStore
+from torch_geometric.data import HeteroData
+from torch_geometric.loader.node_loader import NodeLoader
+from torch_geometric.typing import InputNodes
+
+from graphscope.learning.pyg_neighbor_sampler import PygNeighborSampler
+
+
+class GLTNeighborLoader(NodeLoader):
```

Comment:
NeighborLoader PyG originally in iteration will call filter_fn () to convert the iterative output type (SamplerOutput/HeteroSamplerOutput - > Data/HeteroData), But the output of the PygNeighborSampler iteration is the Data/HeteroData type, so I've removed the parts related to filter_fn() here so that they can be properly connected.